core/src/main/java/org/apache/accumulo/core/iterators/SortedKeyValueIterator.java

```diff
@@ -147,4 +147,16 @@
    *              if not supported.
    */
   SortedKeyValueIterator<K,V> deepCopy(IteratorEnvironment env);
+
+  /**
+   * Called when the client (i.e. calling iterator or the tserver) will support this iterators ability to yield. If called, then when next or seek is called
```

Comment:
Javadoc is best written in third-person, declarative (and should avoid using they word to describe the word being defined), e.g. "Allows implementations to preempt further iteration of this iterator in the current RPC. Implementations can use the `yield` method on the `callback` to instruct the caller to cease collecting more results. The yield method accepts a `Key` which is a hint to the caller to resume the scan without returning more data; that is, the `Key` set should be the last Key processed by this Iterator, either internally or externally that would safely enable this iterator to not return duplicate data or omit data. This feature is not supported for isolated scans. Most iterators do not need to implement this method."